content/www/locales/fr.json

```diff
@@ -2,116 +2,117 @@
-                    "description": "The viewport updates automatically to run your code in real time. Every time you make a change, your code is uploaded and saved so you can resume your work later."
+                    "title": "Edition en temps réel",
+                    "description": "L'apercu se met à jour automatiquement et exécute votre code en temps réel. Dès que vous faites un changement, votre code est sauvegardé en ligne pour que vous puissiez reprendre plus tard."
                 },
                 "feature3": {
-                    "title": "Document Manipulation",
-                    "description": "This is a step above your ordinary word processor or text editor. The online editor features syntax highlighting that is customizable, the ability to undo/redo, a tabbed document interface, keyboard shortcuts, error reporting, and much more."
+                    "title": "Manipulation de documents",
+                    "description": "Au-delà de votre éditeur de texte classique. Notre éditeur inclut une coloration syntaxique personnalisable, une fonction annuler/refaire, une interface en onglets, des raccourcis clavier, le signalement d'erreurs et beaucoup plus."
+			// Seriously?
                 }
             }
         },
         "courses": {
-            "header": "Course Offerings",
-            "subheader": "A Wide Variety to Choose From",
-            "more": "More Courses"
+            "header": "Nos cours",
+            "subheader": "Un large éventail de possibilités",
+            "more": "Plus de cours"
         }
     },
     "catalogue": {
         "pageName": "Catalogue",
         "courses": {
-            "header": "Courses",
-            "subheader": "A listing of all of our available courses"
+            "header": "Cours",
+            "subheader": "Tous nos cours disponibles"
         },
         "paths": {
-            "header": "Pathways",
-            "subheader": "A featured list of available course pathways"
+            "header": "Chemins d'apprentissage",
+            "subheader": "Une liste de chemins d'apprentissage sélectionnés"
         }
     },
     "pricing": {
-        "pageName": "Pricing"
+        "pageName": "Tarifs"
     },
     "about": {
-		"pageName": "About",
+		"pageName": "À propos",
 		"contact": {
-			"header": "Contact Us",
-			"subheader": "Reach out to us about any questions, comments, or concerns.",
+			"header": "Nous contacter",
+			"subheader": "Contactez-nous si vous avez des questions, commentaires, ou préoccupations.",
 			"contactList": {
-				"generalEmail": "General Email",
-				"billingEmail": "Billing Email"
+				"generalEmail": "Email général",
+				"billingEmail": "Email pour la tarification"
 			}
 		},
 		"devTeam": {
-			"header": "The Ideoxan Dev Team",
-			"subheader": "A Diverse Group of Talented Individuals"
+			"header": "L'équipe de développement d'Ideoxan",
+			"subheader": "Un groupe divers d'individus talentueux (et modestes :3)"
```

Comment:
What does `:3` mean?

Reply:
it's an emoticon

Reply:
> it's an emoticon

Please remove any emoticons as well then.

Reply:
Okay.